Service/Sources/Data/DataSource/Remote/API/ClubAttendAPI.swift

```diff
@@ -0,0 +1,91 @@
+import Moya
+import Foundation
+
+enum ClubAttendAPI {
+    case fetchAttendList(clubID: Int, date: String?, period: Period?)
+    case attendanceCreate(clubID: Int)
+    case recordeExcelPrint
+    case changeAttendStatus
+    case statusAllApply
+}
+
+extension ClubAttendAPI: GCMSAPI {
+    var domain: GCMSDomain {
+        return .attend
+    }
+
+    var urlPath: String {
+        switch self {
+        case let .fetchAttendList(clubID, date, period):
```

Comment:
```suggestion
        case let .fetchAttendList(clubID, _, _):
```

사용 안 하는 값이니까 _쓰는 게 더 좋아보여요

Reply:
b413ac4ea3ed45d348fb427835715441511048d6
수정했습니다

Reply:
9cb9f396eaea9da2c80d641dd1f0ea32af935ed2
수정했습니다

Reply:
수정 안됐는데요?

Reply:
74ca19782d52c2702c61f45658e2f3768dc05e5e
수정했습니다

Reply:
fetchAttendList에 clubID는 왜 수정하신 건가요? 사용하게 된건가요?

Reply:
64d1c7e214f737c70c8a99583537587c94bb9574
다시 바꿨습니다